### DISCUSSION TITLE
GoogleMapsOverlay update props

### DISCUSSION THREAD

**Discussion by Simone1315** [visgl/deck.gl#5850] [Q&A]
I am currently using a GeoJsonLayer inside a GoogleMapsOverlay to create a bubble layer over google maps. I would like to change the line width of one of the bubbles when clicking on a particular bubble. At the moment, I am updating the whole overlay each time the user clicks on a bubble, as suggested in this issue :

this.deckOverlay.setProps({ layers: deckLayers });

Is there any way I can partially update the layer without having to create a new layer every time (for example, update a signle prop of a layer?)

**Comment by kylebarron** [comment]
Creating a new deck.gl layer is very cheap: https://deck.gl/docs/developer-guide/using-layers#faq
